scripts/browserstack/teardown_tunnel.sh

```diff
@@ -4,5 +4,17 @@ set -e -o pipefail
 
 
 echo "Shutting down Browserstack tunnel"
-echo "TODO: implement me"
-exit 1
\ No newline at end of file
+
+PID=$(cat $BROWSER_PROVIDER_READY_FILE);
+
+# Resolving the PID from the readyfile.
+kill $PID
+
+
+while [[ -n `ps -ef | grep $PID | grep -v "grep"` ]]; do
+  printf "."
+  sleep .5
+done
```

Comment:
I think it just helps ensuring that the Browserstack tunnel is properly turned off.

Normally the `while`  does not run *once*, because the process terminated synchronously before, but in some situations it might take longer (crashes etc.) and this might help us as well.

> Also added because the Saucelabs teardown script had the same.